agr_java_core/src/main/java/org/alliancegenome/neo4j/entity/node/Gene.java

```diff
@@ -146,4 +147,19 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(primaryKey);
     }
+
+
+    @JsonView(value = {View.AlleleVariantSequenceConverterForES.class})
+    @JsonProperty(value = "crossReferences")
```

Comment:
There is a method in the parent class, GeneticEntity, that has a JsonPropery annotated like this. I am not sure why you need a new method. Does the old one on GeneticEntity not suffice?

Reply:
Adam and I went over a change to this, that will be in the GeneticEntity class... basically 3 methods: `getCrossReferences` which will return the list of neo objects: `getCrossReferenceMap` which will return what the UI is looking for and `getCrossReferenceList` which will return a list of strings of the crossreference "name" field, and is what the search needs. All the JsonProperties go away and we go back to the method names. The UI will need to change to use `crossReferenceMap` in the json but Adam will be putting in a change to that in the UI repo.

Reply:
Sounds good.